server/services/phase.js

```diff
@@ -1,9 +1,105 @@
+/* eslint-disable camelcase */
+// disabling camelcase check so that we can manipulate snake_case attributes without changing structure
+const dayjs = require('dayjs');
+const isBetween = require('dayjs/plugin/isBetween');
 const { dbClient, collections } = require('../db');
+const { getSiteByID } = require('./employers');
+
+dayjs.extend(isBetween);
```

Comment:
This already happened in `client/src/utils/date.js`, does it need to happen here too?

Reply:
Yep, this is server-side. This duplication is one of the reasons why I'd like client and server to be able to share code.

Reply:
AhhI didn't notice they were client/server 👍